src/main/java/com/exasol/containers/wait/LogFileEntryWaitStrategy.java

```diff
@@ -0,0 +1,58 @@
+package com.exasol.containers.wait;
+
+import java.io.IOException;
+
+import org.testcontainers.containers.Container;
+import org.testcontainers.containers.Container.ExecResult;
+import org.testcontainers.containers.ContainerLaunchException;
+import org.testcontainers.containers.wait.strategy.AbstractWaitStrategy;
+
+/**
+ * Strategy that waits for a container to be ready by checking when the language container is unpacked completely.
+ */
+public class LogFileEntryWaitStrategy extends AbstractWaitStrategy {
+    private static final int ONE_SECOND_IN_MILLISECONDS = 1000;
+    private final Container<? extends Container<?>> container;
+    private final String logPath;
+    private final String pattern;
+    private final String logNamePattern;
+
+    /**
+     * Create a new instance of the {@link LogFileEntryWaitStrategy}
+     *
+     * @param container      container in which the log messages reside
+     * @param logPath        path of the log file to search
+     * @param logNamePattern pattern used to find the file name
+     * @param pattern        regular expression pattern for which to look out
+     */
+    public LogFileEntryWaitStrategy(final Container<? extends Container<?>> container, final String logPath,
+            final String logNamePattern, final String pattern) {
+        super();
+        this.container = container;
+        this.logPath = logPath;
+        this.logNamePattern = logNamePattern;
+        this.pattern = pattern;
+    }
+
+    @Override
+    protected void waitUntilReady() {
+        final long expiry = System.currentTimeMillis() + (60 * ONE_SECOND_IN_MILLISECONDS);
```

Comment:
Could you please make this into a constant? I guess, making it `WAIT_DURATION_IN_MILLIS` or similar would be helpful.

Reply:
Done.
Also renamed the polling rate limit.